blocks/shared-styles/intl.json

```diff
@@ -21,28 +21,6 @@
 		"pt": "Vídeo",
 		"sv": "Video"
 	},
-	"byline.by-text": {
```

Comment:
This will mean current blocks (not migrated) will not have any translations for these phrases at the moment

Reply:
Yeah - that's fine by me, but may cause some confusion when a developer comes to do a migration of a block still using code from Shared Styles - technically there are still consumers of this code until the last block is converted

Reply:
added back